cilium/cilium.io
# feat: rss newsletter feed

**This PR:**
- Adds an rss feed for the newsletter page;

**Steps to test**

- Open the [Newsletter page](https://deploy-preview-316--cilium.netlify.app/newsletter/)
- Open the [Newsletter page rss](https://deploy-preview-316--cilium.netlify.app/newsletter/rss.xml)
- Confirm that everything looks and works as expected
